projects/1.18/assets/create-alloyed/alloyed/lang/zh_cn.json

```
@@ -0,0 +1,33 @@
+{
+  "alloyed.ponder.bronze_bell_decoration.header": "青铜钟",
+  "alloyed.ponder.bronze_bell_decoration.text_1": "青铜钟是由青铜制成的装饰方块。",
+  "alloyed.ponder.bronze_bell_decoration.text_2": "用锁链挂起来可以达到很好的效果…",
+  "alloyed.ponder.bronze_bell_decoration.text_3": "…栅栏…",
+  "alloyed.ponder.bronze_bell_decoration.text_4": "…或铁栏杆。",
```

```suggestion
  "alloyed.ponder.bronze_bell_decoration.text_2": "用锁链挂起来可以达到很好的效果……",
  "alloyed.ponder.bronze_bell_decoration.text_3": "……还可以用栅栏……",
  "alloyed.ponder.bronze_bell_decoration.text_4": "……亦或是铁栏杆。",
```
语序变了，就只好这样了，不知道有没有更好的译法